### PR TITLE
test fail When set default time zone to Asia/Tehran

### PR DESCRIPTION
**Pull request by redknight**
Hi
When i set php default time zone to Asia/Tehran 
Test testDeterminesIfCronIsDue #0 has been failed.

### CODE REVIEW

**Comment by DerManoMann** [comment]
On closer inspection it appears that the HoursField change is sufficient for my problem.
cron: '0 */4 * * *'
tz:  'Australia/Adelaide'

File: src/Cron/MinutesField.php [hunk 1]
```diff
@@ -16,8 +16,14 @@ public function increment(\DateTime $date, $invert = false)
     {
         if ($invert) {
             $date->modify('-1 minute');
+            if($date->format('i')=='59'){
+                $date->modify('+1 hour');
```
Comment (mtdowling) [inline src/Cron/MinutesField.php]: Don't you also need to set the minutes back to 0 here?

Comment (DerManoMann) [inline src/Cron/MinutesField.php]: Perhaps, but with 0 as minute value it seems to work.

Also, I wasn't sure what the implications are as there are core tests failing with that change included.

Comment (redknight) [inline src/Cron/MinutesField.php]: I think if set minutes to zero infinite loop was happen

Comment (DerManoMann) [inline src/Cron/MinutesField.php]: Hmm, the cronline given above does work with just the hours class patched, though.

I can see the logic in the minutes class change, but I do not know enough about the internals to understand why that is causing the tests to fail.

**Comment by dragonmantank** [comment]
Hello! We recently pushed out v1.1.0 which includes fixes for non-hourly offsets to timezones such as Asia/Tehran. This may have fixed your issue. 

If it hasn't, please either submit a new PR or rebase your PR against v1.1.0 and re-open this PR.

**Pull request closed without merging**